inference-engine/samples/benchmark_app/main.cpp

```diff
@@ -539,16 +571,19 @@ int main(int argc, char *argv[]) {
             inferRequest->startAsync();
         }
         inferRequestsQueue.waitAll();
-        auto duration_ms = double_to_string(inferRequestsQueue.getLatencies()[0]);
+        duration_ms = double_to_string(inferRequestsQueue.getLatencies()[0]);
         slog::info << "First inference took " << duration_ms << " ms" << slog::endl;
+        auto duration_since_start_ms = double_to_string(get_total_ms_time(runStartTime));
+        slog::info << "Total time for first inference since startup " << duration_since_start_ms << " ms" << slog::endl;
```

Comment:
It should be deleted. We discussed that benchmark_app is not the right place for measurement of time to first inference. We have tests for it. benchmark_app includes many additional steps like configuration, inputs filling, and so on. It will confuse the user.

Reply:
Fixed